src/Umbraco.Web.UI.Client/src/packages/media/media/dropzone/dropzone-media.element.ts

```diff
@@ -52,6 +41,14 @@ export class UmbDropzoneMediaElement extends UmbInputDropzoneElement {
 		document.addEventListener('dragleave', this.#handleDragLeave.bind(this));
 		document.addEventListener('drop', this.#handleDrop.bind(this));
 
+		// TODO: Revisit this. I am not sure why it is needed to call these methods here when they are already called in the constructor of the parent class.
+		// If we do not call them here, the observer will use the wrong instance of the dropzone manager (UmbDropZoneManager instead of UmbMediaDropzoneManager).
```

Comment:
Is this caused because the constructor of the superclass could be invoked before the properties are assigned by the subclass?

You are assigning the instance as a property directly from the subclass:

```js
protected override _manager = new UmbMediaDropzoneManager(this);
```

But the super constructor may already have begun listening on its own instance of the manager? So by calling _observeProgress() again, you are now observing your new instance.

Anyway, it seems you have gotten to the core of the issue, that I ran into when trying this, and you found a solution - kudos!